packages/components/.storybook/preview.tsx

```diff
@@ -1,8 +1,10 @@
+import "@chanzuckerberg/czedi-kit-tokens/css/variables.css";
```

Comment:
what is this doing? or is it setting up for next changes?

Reply:
This adds the css variables to the root - in the long run we would want this tied in with the EDS Global Styles maybe